scripts/build-tags.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Get the value of the LEDGER_ENABLED variable (default to "true" if not set)
+LEDGER_ENABLED="${LEDGER_ENABLED:-true}"
+
+# process build tags
+build_tags=""
+
+if [ "$LEDGER_ENABLED" = "true" ]; then
+	if [ "$OS" = "Windows_NT" ]; then
+		GCCEXE=$(where gcc.exe 2>/dev/null)
+		if [ -z "$GCCEXE" ]; then
+			echo "gcc.exe not installed for ledger support, please install or set LEDGER_ENABLED=false" >&2
+			exit 1
+		else
+			build_tags+="ledger"
+		fi
+	else
```

Comment:
Only necessary if this actually build on windows. Did we verify it did? If not, I would remove this as well.

Reply:
yea i had no idea what this was all about haha I just tried to reproduce whatever it was currently doing. 

This script is also used in the Makefile, so I would leave it in here unless we are specifically calling out in the README that we don't support windows.

Reply:
[not blocking] I don't think we should modify the build tag behavior in the same PR as introducing Goreleaser. Since this code block appears moved from the Makefile, proposal to NOT remove any Windows logic here. If we want to remove Windows logic, that can be done in a distinct follow-up refactor PR.

FWIW the Windows specific logic was added in https://github.com/celestiaorg/celestia-app/pull/2014